core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-page-dropzone/ema-page-dropzone.component.ts

```diff
@@ -45,14 +45,17 @@ export class EmaPageDropzoneComponent {
      * @param {DragEvent} event
      * @memberof EmaPageDropzoneComponent
      */
-    onDrop(event: DragEvent): void {
+    onDrop(event: DragEvent, isEmptyContainer = false): void {
         const target = event.target as HTMLDivElement;
+
         const data: ClientData = JSON.parse(target.dataset.payload);
         const isTop = this.isTop(event);
 
+        const insertPosition = isTop ? 'before' : 'after';
+
         const payload = <PositionPayload>{
             ...data,
-            position: isTop ? 'before' : 'after'
+            position: isEmptyContainer ? undefined : insertPosition
```

Comment:
Do we actually needs this? doesn't look like to me.

Reply:
I got a to clean some parts of this code, by the time I'm focusing on make it work

Reply:
Cleaned!